kafka/src/main/java/com/expedia/adaptivealerting/kafka/StreamsAppConfig.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2018 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.adaptivealerting.kafka;
+
+import com.typesafe.config.Config;
+import lombok.Getter;
+import lombok.val;
+import org.apache.kafka.streams.StreamsConfig;
+
+import java.util.Properties;
+
+import static com.expedia.adaptivealerting.core.util.AssertUtil.notNull;
+
+/**
+ * Kafka streams application configuration. Consolidates both Typesafe Config and Streams Config.
+ *
+ * @author Willie Wheeler
+ */
+public class StreamsAppConfig {
+    
+    /**
+     * Kafka streams configuration key.
+     */
+    private static final String CK_STREAMS = "streams";
+    
+    /**
+     * Inbound topic configuration key.
+     */
+    private static final String CK_INBOUND_TOPIC = "inbound-topic";
+    
+    /**
+     * Outbound topic configuration key.
+     */
+    private static final String CK_OUTBOUND_TOPIC = "outbound-topic";
+    
+    /**
+     * Health status path configuration key.
+     */
+    private static final String CK_HEALTH_STATUS_PATH = "health.status.path";
```

Comment:
I would say let's leave it for now, because there is a good chance that I will reinstate the same health check mechanism after i get the deserialization exception handler working. I would rather just leave it there now as opposed to deleting it from the app and the terraform/k8s configs, only to re-add them. Having said that I will be sure to either use them or delete them.